src/bench/bench.cpp

```diff
@@ -41,18 +42,46 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
-benchmark::BenchRunner::BenchmarkMap& benchmark::BenchRunner::benchmarks()
+namespace benchmark {
+
+std::map<std::string, PriorityLevel> map_priority_level = {
+    std::make_pair("high", PriorityLevel::HIGH),
+    std::make_pair("medium", PriorityLevel::MEDIUM),
+    std::make_pair("low", PriorityLevel::LOW)
+};
+
+PriorityLevel StringToPriority(const std::string& str)
+{
+    if (str == "all") return (PriorityLevel)0xff; // "all" shortcut
+    auto it = map_priority_level.find(str);
+    if (it == map_priority_level.end()) throw std::runtime_error(strprintf("Unknown priority level %s", str));
```

Comment:
Is there a valid concern that the string could be entered in as multicase? Like Low, Medium, High, All?
Maybe this would be better suited in parsePriorityLevel?
```suggestion
    std::transform(str.begin(), str.end(), str.begin(), ::tolower);
    if (str == "all") return (PriorityLevel)0xff; // "all" shortcut
    auto it = map_priority_level.find(str);
    if (it == map_priority_level.end()) throw std::runtime_error(strprintf("Unknown priority level %s", str));
```

Reply:
Something like that would be better suited for the general `ArgsManager::ParseParameters` function. Not only for this new flag.

But I'm not so sure about it. Init flags are keys, and keys shouldn't have different values. Should be a 1-1 map.

Reply:
Makes sense, I was thinking the same thing last night. ArgsManager should perhapse enforce case if that is a good idea. Thanks.